src/raft.rs

```diff
@@ -1691,12 +1691,19 @@ impl<T: Storage> Raft<T> {
                             }
                         }
                     }
-                } else {
+                } else if m.get_from() == INVALID_ID || m.get_from() == self.id {
```

Comment:
Could you please include the comments like in the etcd code? This way we can know why we did this later.

Reply:
done.